FileMapper/config_script.html

```diff
@@ -0,0 +1,362 @@
+<!--
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   https://www.apache.org/licenses/LICENSE-2.0
+   
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<script>
+  // The Browser API key obtained from the Google API Console.
+  // Replace with your own Browser API key, or your own key.
+  var DEVELOPER_KEY = ENVIRONMENT.API_KEY;
+  var DIALOG_DIMENSIONS = {width : 600, height : 425};
+  var pickerApiLoaded = false;
+
+  // Next id for adding a new mapping
+  var nextId = 1;
+  // ID of mapping currently editing
+  var activeId = 0;
+
+  // Config Table Column enumeration
+  var ConfigColumns = {
+    EDIT_BUTTON : 0,
+    LOCAL_PATH : 1,
+    DRIVE_PATH : 2,
+    FILE_OR_FOLDER : 3,
+    DELETE_BUTTON : 4
+  };
+
+  /**
+   * Executed as soon as this page loads
+   * Gets the config data from the server
+   * to display in the table.
+   */
+  $(document).ready(function() {
+    // Display the loading modal 
+    ScriptUtil.displayLoadingMessage("Config Data Is Loading...");
+    google.script.run
+        .withSuccessHandler(function(configData) { displayTable(configData);})
+        .withFailureHandler()
+        .getConfigDataToBuildTable();
+  })
+
+  /**
+   * Loads the Google Picker API.
+   */
+  function callPicker() {
+    gapi.load('picker', {'callback' : function() { pickerApiLoaded = true; }});
+    getOAuthToken();
+  }
+
+  /**
+   * Gets the user's OAuth 2.0 access token from the server-side script so that
+   * it can be passed to Picker. This technique keeps Picker from needing to
+   * show its own authorization dialog, but is only possible if the OAuth scope
+   * that Picker needs is available in Apps Script.
+   */
+  function getOAuthToken() {
+    google.script.run
+        .withSuccessHandler(createPicker)
+        .withFailureHandler()
+        .getOAuthToken();
+  }
+
+  /**
+   * Creates a Picker that can access the user's spreadsheets.
+   *
+   * @param {String} token An OAuth 2.0 access token that lets
+   *     Picker access the file type
+   */
+  function createPicker(token) {
```

Comment:
This method seem to be in all the PRs, can you submit the others and then merge so that it is clear what are the additional changes in this PR?

Reply:
This method is on the client side script hence it is repeated.